### ISSUE TITLE
Weird error when re-exporting abstract class

### ISSUE THREAD

**Issue by wycats** [Microsoft/TypeScript#5483]
I have this in a file `lib/environment.ts`:

```ts
export abstract class Scope<T extends Object> {
  // ...
}
```

And then, in `index.d.ts`, I try to do this:

```ts
export { Scope } from './lib/environment';
```

I get:

```
Module '...' has no exported member 'Scope'
```

I can import the abstract class just fine, and do so in other files. Even more weirdly, TS lets me import this re-export in another file, and properly understands it. It just seems like an erroneous error.

**Comment by DanielRosenwasser** [comment]
Hey @wycats, I'm not able to repro the issue on my end. What OS, version of Node, and version of tsc are you running?

**Comment by wycats** [comment]
I'm seeing the error in VSCode. The version of TypeScript in the tsdk is `1.8.0-dev.20151024`

**Comment by mhegazy** [comment]
i am not able to get repro on VSCode either. i think there can be something else going on here..  do you have a tsconfig.json? do you see this repro on just two the two files above, or do you need to have other files?

**Comment by mhegazy** [comment]
@wycats is this still an issue?

**Comment by mhegazy** [comment]
please reopen if still an issue.
